app/models/form/sales/pages/property_local_authority.rb

```diff
@@ -15,7 +15,7 @@ def questions
     ].compact
   end
 
-  def la_known_question
+  def la_known_question #TODO: Pre 2023, could be removed? Only used here.
     if form.start_date.year < 2023
       Form::Sales::Questions::PropertyLocalAuthorityKnown.new(nil, nil, self)
     end
```

Comment:
A few places where the logic suggests its for forms pre 2023, which can be removed now or not?